models/ens/ens_ethereum_reverse_latest.sql

```diff
@@ -0,0 +1,62 @@
+{{ config(
```

Comment:
I think the model  should be named `ens_reverse_latest`

If you prefer to keep the ethereum part in the name, then there needs to be a modification in the schema added in the dbt_project.yml file

Reply:
ok, I tried to be consistent here with other model definitions where we use `<project>_<chain>_<table>`

Reply:
Should we change the other models to use `ens_ethereum` and adapt the dbt_project.yml, or revert here to `ens_`?

Reply:
yeah I think it was a good idea, I'm actually not sure if ENS is only on the mainnet Ethereum chain ? If that's the case, we can just get rid of ethereum in the name. If not, we should just add it to the dbt_project.yml

Reply:
For now seems to just be on mainnet eth. I'll revert the change.